test/io/test_io.py

```diff
@@ -844,6 +823,7 @@ def test_admin_ready_wo_channel(defaultenv):
         assert response.status_code == 200
 
 
+@pytest.mark.skip
```

Comment:
This test now fails, because we no longer flush the pool on `SIGUSR1`.

The fact that this test relies on that behaviour seems to rather be an implementation detail of the test -- it just needs schema cache reloading to fail, which we used to achieve by setting the session timeout.

Could there be a way to get the introspection query to fail more directly, by messing up the schema or permissions?

Reply:
I think it might be possible by revoking privileges on one of the pg_catalog objects.

Reply:
It works again in its original form now that we flush the pool properly.